charts/c11h-basic/templates/service.yaml

```diff
@@ -17,6 +20,11 @@ spec:
       protocol:  {{ .protocol | default "TCP" }}
       name: {{ .name }}
   {{- end }}
+  {{- if .Values.service.metrics.enabled -}}
```

Comment:
@gypsydiver  : I think the possibility to re-use an existing service port (labelled web or webmetrics) is not covered.
When service.metrics.enabled is set, always the metrics port is created. That might lead to a port clash.